exporter/jaeger/jaeger.go

```diff
@@ -59,6 +59,9 @@ type Options struct {
 
 	// ServiceName is the Jaeger service name.
 	ServiceName string
+
+	// Tags are added to Jaeger Process exports
+	Tags []Tag
```

Comment:
How about calling this `ProcessTags` to distinguish them from tags associated with attributes on Spans?

Reply:
Thought about that too, but then I sticked with the way it is defined in Jaeger: https://github.com/jaegertracing/jaeger-idl/blob/b59e84ebb35cd858614f836e47540d6f95e0cf88/thrift/jaeger.thrift#L68
But no big deal to change that!

Reply:
I think ProcessTags is clearer since Tags might be confused with the tags package the context of OpenCensus.